### PR TITLE
Assignmend 2

### PR DESCRIPTION
**Pull request by JustAntoRS**

### CODE REVIEW

File: Assignment2/JustAntoRS-AntonioRamirez.ttl [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+
+:Class01 :includes :Sensor029,
+         :includes :Computer101.
+
+:Sensor029 :hasMeasurement :Measurement8401.
+
+:Computer101 :hasOwner :User10A.
+
+:Measurement8401 :hasTemperature : "29":^^xsd:integer.
```
Comment (ocorcho) [inline Assignment2/JustAntoRS-AntonioRamirez.ttl:12]: Wrong syntax. This does not parse.

File: Assignment2/JustAntoRS-AntonioRamirez.ttl [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+:Measurement8401 :atTime :"2010-06-12T12:00:12"^^xsd:dateTime.
```
Comment (ocorcho) [inline Assignment2/JustAntoRS-AntonioRamirez.ttl:13]: Same here.

File: Assignment2/JustAntoRS-AntonioRamirez.ttl [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+:User10A :hasName :"Pedro"^^xsd:string.
```
Comment (ocorcho) [inline Assignment2/JustAntoRS-AntonioRamirez.ttl:15]: Same here.

**Review by ocorcho — CHANGES_REQUESTED** [review]

File: Assignment2/JustAntoRS-AntonioRamirez.ttl [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+
+:Class01 :includes :Sensor029;
+         :includes :Computer101.
+
+:Sensor029 :hasMeasurement :Measurement8401.
+
+:Computer101 :hasOwner :User10A.
+
+:Measurement8401 :hasTemperature "29":^^xsd:integer;
```
Comment (ocorcho) [inline Assignment2/JustAntoRS-AntonioRamirez.ttl:12]: Please, parse it, and you will see that there is an error.

**Review by ocorcho — CHANGES_REQUESTED** [review]

File: Assignment2/JustAntoRS-AntonioRamirez.ttl (hunk 4 above)
Comment (JustAntoRS) [reply Assignment2/JustAntoRS-AntonioRamirez.ttl]: Done, sorry for all the iterations on this :sweat_smile:

**Pull request merged**